### PR TITLE
chore(artifact_proxy): add shorebirdtech/engine@7aa5c44764e to mappings

### PR DESCRIPTION
**Pull request by eseidel**
I also deleted the rust calls from bootstrap.sh (since they failed for me) and removed build_engine.sh since it's now done differently in shorebirdtech/build_engine

### CODE REVIEW

File: packages/artifact_proxy/lib/config.dart [hunk 1]
```diff
@@ -8,62 +8,50 @@ const config = ProxyConfig(
   },
 );
 
+// On the assumption artifact layouts don't change very often for Flutter.
+class _EngineMapping37 extends EngineMapping {
```
Comment (felangel) [inline packages/artifact_proxy/lib/config.dart]: Is the 37 supposed to represent Flutter 3.7.x?

Comment (eseidel) [reply packages/artifact_proxy/lib/config.dart]: Yeah. I tried 3_7 and the linter got mad at me.

Comment (bryanoltman) [reply packages/artifact_proxy/lib/config.dart]: Maybe `_EngineMapping3_7` instead?

Comment (eseidel) [reply packages/artifact_proxy/lib/config.dart]: Will change and update the linter. :)

Comment (bryanoltman) [reply packages/artifact_proxy/lib/config.dart]: Sniped 🙂 

I'd vote to override the linter on this one.

Comment (eseidel) [reply packages/artifact_proxy/lib/config.dart]: Overriden. 🤣  PTAL.

**Review by bryanoltman — APPROVED** [review]

**Pull request merged**